addons/api/mirage/factories/host-catalog.js

```diff
@@ -53,7 +53,8 @@ export default factory.extend({
   worker_filter: function () {
     if (
       this.type === TYPE_HOST_CATALOG_DYNAMIC &&
-      this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_AWS
+      this.plugin?.name === TYPE_HOST_CATALOG_PLUGIN_AWS &&
```

Comment:
Do the tests work with this? Shouldn't it be a `||` since this you'll never have a name that's both aws and gcp?

Reply:
yeah the tests work because we only check for the label and not for the content in it. 

I can update the tests to include the content and fix this condition

Reply:
Ah gotcha, I don't think we need to check the contents, I see that that the test was just checking to see if the field was present which is good enough